docs/configure-statperf.md

```diff
@@ -0,0 +1,103 @@
+# StatPerf Collector
+
+StatPerf collects performance metrics from ONTAP by invoking the ONTAP CLI command via the private Rest CLI. In this case, the ONTAP CLI command used is:
```

Comment:
StatPerf collects performance metrics from ONTAP by invoking the ONTAP CLI statistics command via the private Rest CLI. The full ONTAP CLI command used is: